doc/man/man3/notcurses_visual.3.md

```diff
@@ -343,10 +345,11 @@ or if both ***nc*** and ***n*** are **NULL**.
 
 **ncvisual_media_defblitter** returns the blitter selected by **NCBLIT_DEFAULT**
 in the specified configuration. If UTF8 is not enabled, this will always be
-**NCBLIT_1x1**. If ***scale*** is **NCSCALE_NONE** or **NCSCALE_SCALE**, the
-aspect-preserving **NCBLIT_2x1** will be returned. If sextants are available
-(see **notcurses_cansextant**), this will be **NCBLIT_3x2**, or otherwise
-**NCBLIT_2x2**.
+**NCBLIT_1x1**. If octants are available, (see **notcurses_canictant**), the
```

Comment:
`canoctant` i think you mean. and i don't think `4x2` is aspect-preserving? am i being stupid?

the whole reason why `2x1` preserves aspect (mostly) is that a cell is (usually) (roughly) twice as tall as it is wide. so when you put two pixels in there, on top of one anotther, you suddenly have pixels be squares, like they (usually) are in your image. but, `4x2` is a linear map from `2x1`, so yeah, i guess that works too. good deal. nice to have another aspect-preserving blitter!

i'm not sure about it being the default for `NCSCALE_NONE` and `NCSCALE_SCALE`, though. `2x1` allows full color fidelity. remember, you only have a background and foreground for each cell. so with two source pixels mapped to the cell, you can do both colors losslessly. there's no way to losslessly pack 8 pixels into 2 cell colors, though. so you're going to lose some image quality for certain images, and ruin pathological ones.

so i think we stick with `2x1`.

now, maybe it *should* be the default for `NCSCALE_STRETCH` (which is currently `3x2`, which is honestly a garbage blitter).

Reply:
I don't quite follow the reasoning. The `4x2` aspect-preserving mode is never chosen when the caller wants to preserve the aspect, but it might be chosen when the caller explicitly allows non-aspect preserving mapping? That seems a bit confusing.

I'd be fine with never making `4x2` the default (at the moment it's new and untested). In the future, maybe there could be a new flag to preserving colour accuracy?

Reply:
i'm thinking of octants as basically a more attractive, predictable braille. i.e. both are just implementing a 4x2 blitter, but you pretty much know what octants are going to look like. so octants have the same advantages and disadvantages otherwise that braille does -- the advantage is that you can get higher resolution in a fixed cell geometry, and the disadvantage is that you stand to lose color fidelity (which can in certain cases make the image effectively unrecognizable).

4x2 is chosen when the user requests 4x2, whether they choose `_SCALE`, `_NONE`, or `_STRETCH`. this only affects the case where they choose `NCBLIT_DEFAULT` together with `_SCALE` or `_NONE`. if you choose 4x2 explicitly, you get 4x2 (unless we can't do it). does that make sense?

Reply:
Yes, I agree.